vng_api_common/serializers.py

```diff
@@ -18,41 +17,29 @@
 
 try:
     # 1.1.x
-    from relativedeltafield.utils import format_relativedelta, relativedelta
+    from relativedeltafield.utils import format_relativedelta, parse_relativedelta
 except ImportError:
     try:
         # 1.0.x
-        from relativedeltafield import format_relativedelta, relativedelta
+        from relativedeltafield import format_relativedelta, parse_relativedelta
     except ImportError:
         format_relativedelta = None
-        relativedelta = None
+        parse_relativedelta = None
 
 
 class DurationField(fields.DurationField):
```

Comment:
Where in Open Zaak is this field used actually? I can't find any part of the code where it's imported